wurst/_items/Kits/ForgeKit.wurst

```diff
@@ -0,0 +1,51 @@
+package ForgeKit
+
+import ItemObjEditing
+import AbilityObjEditing
+import ObjectIds
+// import PublicLibrary
+import ID
+
+let ABILITY_ID    = compiletime(ABIL_ID_GEN.next())
+let ITEM_ID       = compiletime(ABIL_ID_GEN.next())
+let BASE_ITEM_ID  = 'rst1'
+let MODEL         = "Doodads\\LordaeronSummer\\Terrain\\Barricade\\Barricade0.mdx"
```

Comment:
Do not reference string constants like this directly. Either use the `Assets` package in the standard library or the `LocalAssets` package I recently added for our own imports. The `LivingClay` package demonstrats both.

Reply:
We don't have `ItemIds` in the standard library, so please add that to `LocalAssets`, even though it technically isn't local.